tests/settings/SettingTest.py

```diff
@@ -34,8 +34,8 @@ def test_path(self):
         self.assertEqual(path(self.uut),
                          os.path.abspath(os.path.join('.', '22')))
 
-        abspath = os.path.abspath('.')
-        self.uut = Setting('key', re.escape(abspath))
+        abspath = PathArg(os.path.abspath('.'))
+        self.uut = Setting('key', PathArg(abspath))
```

Comment:
`abspath` is already `PathArg()`'d. No need to do it again in `Setting` instantiation